library/cwm/src/modules/CWM.rb

```diff
@@ -1026,6 +1060,30 @@ def validate_current_widgets(event)
     publish function: :ShowAndRunOrig, type: "symbol (list <string>, map <string, map <string, any>>, term, string, string, string, map)"
     publish function: :InitNull, type: "void (string)"
     publish function: :StoreNull, type: "void (string, map)"
+
+  private
+
+    def widgets_in_contents(contents)
+      contents.each_with_object([]) do |arg, res|
+        case arg
+        when ::CWM::AbstractWidget then res << arg
+        when Yast::Term then res.concat(widgets_in_contents(arg))
```

Comment:
"Undefined method `each_with_object` in Yast::Term" when I run examples/object_api.rb